src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,178 @@
+package duke;
+
+import java.io.*;
```

Comment:
Perhaps it would be better to specify exactly which imports you are using instead of using the wildcard?